### DISCUSSION TITLE
Orphaned zettels and folgezettel hierarchy generation for flat zettelkasten

### DISCUSSION THREAD

**Discussion by jazmit** [srid/emanote#449] [Ideas]
I just migrated from neuron, and immediately there is a feature I really miss - I keep my zettel in a flat format with no sub-folders, which has several advantages, but one disadvantage is that it's difficult to find orphaned zettels that are not in the folgezettel hierarchy. Neuron would generate the hierarchy from the folgezettel links and explicitly display orphans to allow me to connect them up to other zettels.  Is there a way to do this with emanote?

**Comment by srid** [comment]
Related: https://github.com/srid/emanote/discussions/64
